### ISSUE TITLE
Title rendering of encoded characters

### ISSUE THREAD

**Issue by peterdavidhamilton** [DFE-Digital/register-early-career-teachers-public#2264]
## Estimated priority

Medium 🏃- we should fix this in the sprint

## What is happening?

Encoded characters are appearing in the page head

## What should be happening?

`&` should render correctly in the browser title or tab

## Steps to replicate the issue

Select an appropriate body, school or other entity, with a special character in their name.

## Screenshots, videos or other things that might help us fix the bug

NB: very top of screenshots

<img width="1220" height="436" alt="Image" src="https://github.com/user-attachments/assets/75a4a9d0-8d9f-4cae-a218-e3462afe186f" />

<img width="1219" height="676" alt="Image" src="https://github.com/user-attachments/assets/b9659517-c2d8-42b6-9252-6994a08c42ac" />
